code/game/objects/items/weapons/defib.dm

```diff
@@ -347,7 +347,7 @@
 			H.Weaken(5)
 			playsound(get_turf(src), 'sound/machines/defib_zap.ogg', 50, 1, -1)
 			H.emote("gasp")
-			if(!H.undergoing_cardiac_arrest() && (prob(10) || defib.combat)) // Your heart explodes.
+			if(!H.undergoing_cardiac_arrest() && prob(10)) // Your heart explodes.
```

Comment:
Very minor nitpick, but this would likely be more performance-friendly if the procs were swapped around:
```suggestion
			if(prob(10) && !H.undergoing_cardiac_arrest()) // Your heart explodes.
```
That way it only checks `H.undergoing_cardiac_arrest()` 10% of the time, rather than checking every time and having 90% of them disregarded.